client/src/components/rounds/Apply.tsx

```diff
@@ -125,6 +128,15 @@ function Apply() {
     };
   }, [props.applicationState]);
 
+  useEffect(() => {
+    if (
+      props.applicationState?.status === ApplicationStatus.Error ||
+      props.applicationError
+    ) {
+      toggleStatusModal(false);
+    }
+  }, [props.applicationStatus, props.applicationError]);
```

Comment:
I think here we can just pass `[props.applicationStatus]` and then without checking we can directly do `toggleStatusModal(props.applicationStatus === ApplicationStatus.Error)`. wdyt @codenamejason ?

Reply:
I tried this and it would flash the StatusModal and then nothing.